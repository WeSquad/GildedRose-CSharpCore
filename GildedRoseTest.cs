using Xunit;
using System.Collections.Generic;
using ApprovalTests.Reporters;
using ApprovalTests;
using System;

namespace csharpcore
{
    [UseReporter(typeof(DiffReporter))]
    public class GildedRoseTest
    {
        [Fact]
        public void FooItemTest()
        {
            IList<Item> Items = new List<Item> { new Item { Name = "foo", SellIn = 0, Quality = 0 } };
            GildedRose app = new GildedRose(Items);
            app.UpdateQuality();
            Assert.Equal("foo", Items[0].Name);
        }

        [Fact]
        public void TestUseOfApprovalTests()
        {
            var names = new[] { "Llewellyn", "James", "Dan", "Jason", "Katrina" };
            Array.Sort(names);
            Approvals.VerifyAll(names, "");
        }
    }
}