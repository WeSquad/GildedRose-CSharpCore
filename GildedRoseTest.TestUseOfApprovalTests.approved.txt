[foo,0,0] => Name:foo, SellIn:-1, Quality:0
